src/tools/miri/src/machine.rs

```diff
@@ -262,11 +263,15 @@ pub struct AllocExtra {
     /// Weak memory emulation via the use of store buffers,
     ///  this is only added if it is enabled.
     pub weak_memory: Option<weak_memory::AllocState>,
+    /// A backtrace to where this allocation was allocated,
+    ///  as this is recorded for leak reports it only exists
+    ///  if this allocation is leakable.
```

Comment:
```suggestion
    /// A backtrace to where this allocation was allocated.
    /// As this is recorded for leak reports it only exists
    /// if this allocation is leakable.
```

Reply:
I was following the format of the doc comments for the other members, so I tweaked those too.